### ISSUE TITLE
Slug titles frontend

### ISSUE THREAD

**Issue by alejandri** [mrakodol/Laravel-5-Bootstrap-3-Starter-Site#143]
Hi great work. How i can put slug titles in frontend. Any Idea. Thanks

**Comment by stojankukrika** [comment]
I do not know what did you need.Can you explain?

**Comment by alejandri** [comment]
Yes. When open one article of news in this url format (http://localhost/mrakodol/news/21) should be http://localhost/mrakodol/news/title-of-my-article,  I want to ask you some idea to implement
